libraries/AP_Notify/OreoLED_PX4.h

```diff
@@ -74,10 +110,73 @@ class OreoLED_PX4 : public NotifyDevice
         uint8_t red;
         uint8_t green;
         uint8_t blue;
+        uint8_t amplitude_red;
+        uint8_t amplitude_green;
+        uint8_t amplitude_blue;
+        uint16_t period;
+        int8_t repeat;
+        uint16_t phase_offset;
+
+        inline oreo_state() {
+            clear_state();
+        }
+
+        inline void clear_state() {
```

Comment:
Will do. Also, l'll fix the byte ordering on those uint16 values too. Never realized (until I looked at the constants you'd added) that I had the byte order backwards. (duh!)